.clang-tidy

```diff
@@ -17,6 +17,7 @@ Checks: >
     -bugprone-incorrect-enable-if,
     -bugprone-switch-missing-default-case,
     -bugprone-empty-catch,
+    -bugprone-unchecked-optional-access,
```

Comment:
This seems to have too much false positives - at least I couldn't understand why it fires in vast majority of places where we have it now...